contrib/test/integration/build/kubernetes.yml

```diff
@@ -2,7 +2,7 @@
 
 - name: clone kubernetes source repo
   git:
-    repo: "https://github.com/runcom/kubernetes.git"
+    repo: "https://github.com/mrunalp/kubernetes.git"
```

Comment:
I'ze confused.  How does changing the repo from runcom's to yours fix logging?  Should/can we set up a repor under ProjectAtomic for this rather than under a personal repo?

Reply:
@TomSweeneyRedHat because the branch in my fork has a commit on top that I mentioned in the first comment https://github.com/mrunalp/kubernetes/commit/945f530782dbf4c5069a4df9bbe6d6e60284d44d

Reply:
Oops!  Sorry @mrunalp, I skimmed right by it.  The changes there LGTM.  My question still holds if it makes sense to have this under ProjectAtomic though.  Something to consider.

LGTM

Reply:
No Tom, our forks are just fine as there's no plan to have something pseudo official in projectatomic.